### DISCUSSION TITLE
Laravel 10 and PHPUnit 10

### DISCUSSION THREAD

**Discussion by wverhoogt** [wintercms/winter#968] [Ideas]
I am working on a plugin to use Livewire in WinterCMS. Livewire requires Laravel 10 so...
I did some work trying to find out how much changes this would need, and found out that the large majority of changes are in the tests.
There might be quite a few repo´s which will require updates, possibly much work. Would it be a good idea if I prepare PR´s for this changes?

**Comment by LukeTowers** [comment]
Sure @wverhoogt, I've tried it out before and also found that not many changes were required but haven't had the time yet to flip the switch so to speak

**Comment by wverhoogt** [comment]
I am mostly done with the change. I ran into a few interesting things, but nothing too problematic. I noticed that the winter:test console command documentation is no longer correct (the --core argument has been replaced by  --module and the --bootstrap option is new).
I can´t create a correct composer.json for the main winter repository, because I don´t know the versions which will be assigned to the cms, system and backend modules and the storm library. I am testing with private repositories. Commit it anyway?

**Reply by LukeTowers** [reply]
Yes please, I can help out with that aspect of it. If you'd be willing to submit PRs to fix the incorrect documentation that would be appreciated too!

**Comment by wverhoogt** [comment]
I´ve created pull requests for winter and storm. Especially for me these are big PR´s. 
I will take a look at the documentation.

**Comment by wverhoogt** [comment]
Do you have time to check my PR´s?

**Reply by josephcrowell** [reply]
Did you pull in Benno's Laravel 9 compatibility revision?

**Comment by wverhoogt** [comment]
Yes, I have.
